lib/max/other/lib-acl.inc.php

```diff
@@ -227,6 +262,53 @@ function MAX_AclSave($acls, $aEntities, $page = false)
     return true;
 }
 
+function MAX_AclDeleteValues($aclsTable, $fieldId, $aclsObjectId)
+{
+    $doAcls = OA_Dal::factoryDO($aclsTable);
+    $doAcls->whereAdd($fieldId.' = '.$aclsObjectId);
+    return $doAcls->delete(true);
+}
+
+function MAX_AclAddValues($acls, $aclsTable, $fieldId, $aclsObjectId)
+{
+    if (!empty($acls))
+    {
```

Comment:
Code style, unless it's a class/function/etc definition, the opening curly brace should be on the previous line. The file has probably a mixed style already, but since you're touching it, it's better to fix the parts that are "broken".

Reply:
Sure - file cleaned up now.